### PR TITLE
Duplicate of PR #8

### PR DESCRIPTION
**Pull request by chemistry-sourabh**
Created Duplicate of PR #8 since my fork link was gone due to private to public conversion.

Please continue discussions here. I am closing old PR

### CODE REVIEW

**Comment by chemistry-sourabh** [comment]
I renamed test to tests. Have a look.

Also @xuhang57  please re approve if everything is ok.

**Comment by xuhang57** [comment]
working on the ci. let's get that merged first.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,209 @@
+
+# Created by https://www.gitignore.io/api/vim,linux,macos,emacs,windows,pycharm
+
```
Comment (Izhmash) [inline .gitignore]: A few suggestions for files we might want to add: `*.pyc`, `*.egg`, `*.egg-info`, `.venv`, `.coverage`

**Comment by chemistry-sourabh** [comment]
Added Python in gitignore.io and updated the gitignore with resultant file.
Rebased on top of working CI.

**Pull request merged**